pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-271_GROUP_CLINICAL_NOTES" author="pegah">
+        <sql>
+            SELECT @realm:= ddp_instance_id FROM ddp_instance WHERE instance_name='singular';
+            SELECT @orderNumber := order_number FROM dev_dsm_db.field_settings where ddp_instance_id = 32 and column_display = 'Protein-losing enteropathy' and
+            field_type = 'GROUP_CLINICAL_NOTES' ;
+
+            update field_settings set order_number = order_number + 2
+            where field_settings_id in ( select field_settings_id from (select * from field_settings) as tmp
+            where field_type = 'GROUP_CLINICAL_NOTES' and order_number >= @orderNumber) and field_settings_id &lt;&gt; 0;
+
+            INSERT INTO field_settings (ddp_instance_id, field_type, column_name, column_display, display_type, order_number,
+            possible_values )
+            SELECT ddp_instance_id , 'GROUP_CLINICAL_NOTES', 'PRIOR_LIVER_TRANSPLANT' ,'Prior listing for liver transplant',
+            'RADIO', @orderNumber,  '[{"value":"YES","name":"Yes"},{"value":"NO","name":"No"},{"value":"UNKNOWN","name":"Unknown"}]'
+            from ddp_instance where instance_name = 'Singular';
```

Comment:
instance_name here and in L27 `Singular` up in L9 `singular`
does it matter?

Reply:
No I don't think SQL cares for VARCHAR but I will update it